Department.java

```diff
@@ -0,0 +1,3 @@
+public class Department {
```

Comment:
I noticed that this class has been moved to a package, however it is still part of the changes. Your last commit did not remove this class. This is how I see the code in your branch right now:

```
C:\DATA\rock-n-roll-payroll>tree \DATA\rock-n-roll-payroll /f /a
Folder PATH listing for volume Windows
Volume serial number is A42C-7141
C:\DATA\ROCK-N-ROLL-PAYROLL
|   Department.java
|   DepartmentManager.java
|   Employee.java
|   HourlyEmployee.java
|   Management.java
|   README.md
|   SalaryEmployee.java
|   StoreManager.java
|   StoreOperators.java
|   Task.java
|   TaskEmployee.java
|
+---lib
|       Department.java
|       Task.java
|
+---personnel
|       DepartmentManager.java
|       Employee.java
|       HourlyEmployee.java
|       Management.java
|       SalaryEmployee.java
|       StoreManager.java
|       StoreOperators.java
|       TaskEmployee.java
|
\---tst
        DepartmentManagerTest.java
        EmployeeTest.java
        HourlyEmployeeTest.java
        SalaryEmployeeTest.java
        StoreManagerTest.java
        TaskEmployeeTest.java
```

Reply:
How can I remove the files that are not sorted into folders?  On my pc I don't see them anymore since they have been sorted. Is this something where I would have to "untrack" or delete them? if so how do I go about doing that?

Reply:
Can you post here the output of `git status` on this branch? Literally copy & paste please

Reply:
```
calan@LAPTOP-7EA1N3GG MINGW64 /d/Dev/MyJava/rock-n-roll-payroll (feature/implement-skeleton)
$ git status
On branch feature/implement-skeleton
Your branch is up to date with 'origin/feature/implement-skeleton'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        deleted:    README.md

Untracked files:
  (use "git add <file>..." to include in what will be committed)

        lib/Department.class
        lib/Task.class
        personnel/DepartmentManager.class
        personnel/Employee.class
        personnel/HourlyEmployee.class
        personnel/Management.class
        personnel/SalaryEmployee.class
        personnel/StoreManager.class
        personnel/StoreOperators.class
        personnel/TaskEmployee.class
        tst/DepartmentManagerTest.class
        tst/EmployeeTest.class
        tst/HourlyEmployeeTest.class
        tst/SalaryEmployeeTest.class
        tst/StoreManagerTest.class
        tst/TaskEmployeeTest.class

no changes added to commit (use "git add" and/or "git commit -a")

calan@LAPTOP-7EA1N3GG MINGW64 /d/Dev/MyJava/rock-n-roll-payroll (feature/implement-skeleton)
$
```